jupyter_collaboration/loaders.py

```diff
@@ -177,7 +182,10 @@ async def notify(self) -> None:
         async with self._lock:
             path = self.path
             # Get model metadata; format and type are not need
-            model = await ensure_async(self._contents_manager.get(path, content=False))
+            try:
+                model = await ensure_async(self._contents_manager.get(path, content=False))
+            except Exception as e:
+                self._log.error(f"Error loading metadata from file: {self.path}\n{e!r}", exc_info=e)
```

Comment:
What happens if there is an exception? If will fail later because `model` is not defined.